AFQ/tasks/data.py

```diff
@@ -449,6 +451,103 @@ def gq_ai(gq_params):
     return AI, dict(GQParamsFile=gq_params)
 
 
+@pimms.calc("rumba_model")
+def rumba_model(gtab,
+                rumba_wm_response=[0.0017, 0.0002, 0.0002],
+                rumba_gm_response=0.0008,
+                rumba_csf_response=0.003,
+                rumba_n_iter=600):
+    """
+    fit for RUMBA-SD model as documented on dipy reconstruction options
+
+    Parameters
+    ----------
+    rumba_wm_response: 1D or 2D ndarray or AxSymShResponse.
+        Able to take response[0] from auto_response_ssst.
+        default: array([0.0017, 0.0002, 0.0002])
+    rumba_gm_response: float, optional
+        Mean diffusivity for GM compartment.
+        If None, then grey matter volume fraction is not computed.
+        Default: 0.8e-3
+    rumba_csf_response: float, optional
+        Mean diffusivity for CSF compartment.
+        If None, then CSF volume fraction is not computed.
+        Default: 3.0e-3
+    rumba_n_iter: int, optional
+        Number of iterations for fODF estimation.
+        Must be a positive int.
+        Default: 600
+    """
+    return RumbaSDModel(
+        gtab,
+        wm_response=np.asarray(rumba_wm_response),
+        gm_response=rumba_gm_response,
+        csf_response=rumba_csf_response,
+        n_iter=rumba_n_iter,
+        recon_type='smf',
+        n_coils=1,
+        R=1,
+        voxelwise=False,
+        use_tv=False,
+        sphere=default_sphere,
+        verbose=True)
+
+
+@pimms.calc("rumba_params")
+@as_file(suffix='_odfmodel-RUMBA_desc-diffmodel_dwi.nii.gz')
+@as_img
+def rumba_params(rumba_model, data, brain_mask):
+    """RUMBA SHM Parameters"""
+    rumba_fit = rumba_model.fit(
+        data,
+        mask=nib.load(brain_mask).get_fdata())
+    odf = rumba_fit.odf(sphere=default_sphere)
+    rumba_shm, _, _ = extract_odf(odf)
+    meta = dict()
+    return rumba_shm, meta
+
+
+@pimms.calc("rumba_fit")
+def rumba_fit(rumba_model, rumba_params):
+    """RUMBA FIT"""
+    return RumbaFit(
+        rumba_model,
+        nib.load(rumba_params).get_fdata())
+
+
+@pimms.calc("rumba_f_csf")
+@as_file(suffix='_odfmodel-RUMBA_desc-CSF_dwi.nii.gz')
```

Comment:
Maybe use `probseg` as suffix? https://bids-specification.readthedocs.io/en/stable/glossary.html#probseg-suffixes

Reply:
@arokem if i’m understanding the docs correctly, each filename can only have one suffix after all of the keyword-value pairs, so the correct name would then be 

`@as_file(suffix='_odfmodel-RUMBA_desc-CSF_probseg.nii.gz’)`?

Reply:
Thats correct @chiuhoward !

Reply:
I think its correct! I just use `dwi` when I don't have anything else to put.

Reply:
So `probseg` is better here

Reply:
Sorry one last question @36000, `probseg` should be the suffix not just for `@pimms.calc("rumba_f_gm”)` but also for `@pimms.calc("rumba_f_csf”)`, `@pimms.calc("rumba_f_wm”)` and `@pimms.calc("rumba_params”)`?